docs/index.rst

```diff
@@ -7,10 +7,18 @@ The `ops` library is a Python framework for writing and testing Juju charms.
 
 The library (`available on PyPI`_) provides:
 
-- :ref:`ops_main_entry_point`, used to initialise and run your charm;
 - :ref:`ops_module`, the API to respond to Juju events and manage the application;
+- :ref:`ops_main_entry_point`, used to initialise and run your charm;
 - :ref:`ops_pebble_module`, the Pebble client, a low-level API for Kubernetes containers;
-- :ref:`ops_testing_module` frameworks for unit testing charms in a simulated environment;
+- the APIs for unit testing charms in a simulated environment:
+
+  - :doc:`State-transition testing </state-transition-testing>`. This is the
+    recommended approach.
+  - :doc:`Harness </harness>`. This is a deprecated framework, and has issues,
+    particularly with resetting the charm state between Juju events. It will be
+    moved out of the base ``ops`` install in an ops release in the future. Charm
```

Comment:
```suggestion
    moved out of the base ``ops`` install in an ops release in the future; at that point, charm
```

Reply:
It also seems like too much information for this point in the docs. Can we move everything from "It will be moved out..." to the Harness page?

Reply:
Happy to move it - in terms of "at that point": I updated the description to include this but didn't mention it in a comment: I added the ability to do `pip install ops[harness]` now (this was in the spec, but I had forgotten). This eases transition in the future, because charms can use `pip install ops[harness]` now and it should seamlessly work when it becomes separated.